assets/blocks/course-outline/outline-block/existing-lessons-modal/lessons-modal-style.scss

```diff
@@ -0,0 +1,102 @@
+.wp-block-sensei-lms-course-outline__existing-lessons-modal {
```

Comment:
One more suggestion I noticed here. But not critical, so if you want to ignore to merge it earlier, maybe we could improve it in a future refactor.

Maybe we could leave the `wp-block-sensei-lms-course-outline__` prefix out of this class, and start a new block here. In order to follow the BEM standards, so we have a block, and element, and sometimes a modifier.

Otherwise, we start creating a big hierarchy having huge names. 🤭

Reply:
Thanks, that was especially helpful, as I had to review all the classes. I had mistakes and typos there. It also fixed the other issue you pointed at, with content scrolling.
Fixed here: https://github.com/Automattic/sensei/pull/7368/commits/9e1d6ed78fafd4d2e4e94b51bfd935e3d66fa3d5